src/Client.js

```diff
@@ -169,6 +172,7 @@ function Client(options) {
     headers: {},
     fetch: undefined,
     queryTimeout: null,
+    http2SessionIdleTime: 500,
```

Comment:
Have you read Ewan's comment on my PR? https://github.com/fauna/faunadb-js/pull/460#issuecomment-813687728

What do you think? Can we also allow setting the default idle time by environment variable?